src/controllers/CommentsController.js

```diff
@@ -52,6 +52,101 @@ class CommentsController {
       return res.status(response.code).json(response);
     }
   }
+
+  /**
+   * @static
+   * @param {Object} req
+   * @param {object} res
+   * @return {object} comment
+   */
+  static async getAllComments(req, res) {
+    const { lastId } = req.query;
+    const { artId } = req.params;
+    const allComments = await Comment.findAll({ where: { artId } });
+    const lastCommentId = allComments[allComments.length - 1].id;
+    let retriveOffset = lastCommentId - (lastId - 1);
+    if (retriveOffset < 0 || retriveOffset >= lastCommentId) retriveOffset = 0;
+    const limit = 10;
+    const offset = retriveOffset;
+    try {
+      const getComments = await Comment.findAll({
```

Comment:
I think there should be a better way of doing this than hitting the database twice

Reply:
I will research on this

Reply:
This seems like the only way I could find as I needed the total number of existing comments in the database to be able to do the calculations I need for the offset in the paginated query.